src/order/patch-order.ts

```diff
@@ -0,0 +1,110 @@
+import { fetch } from "undici";
```

Comment:
Looks like `undici.fetch` only works on Node 16.8 or later:

https://www.npmjs.com/package/undici#undicifetchinput-init-promise

Seems like you get an error on earlier versions of Node:

```
callback: 'onError', errorObject: Error: createOrder callback failed - (0 , undici_1.fetch) is not a function
    at http://127.0.0.1…}
```

might be worth adding an `.nvmrc` or calling this out in the Readme

Reply:
Great point @dtjones404! would you be up for sending a PR to add this? 

If we want to get fancy, we could enforce it with a semver check similar to what we do in checkout-components with this script helper: https://github.com/paypal/paypal-checkout-components/blob/main/scripts/check-node-version.js